scss/core/_utilities.scss

```diff
@@ -72,9 +72,34 @@ $color-levels: 100, 200, 300, 400, 500, 600, 700, 800, 900;
 }
 
 .mis-auto {
-  margin-inline-start: auto;
+  margin-inline-start: auto !important;
 }
 
 .mie-auto {
-  margin-inline-end: auto;
+  margin-inline-end: auto !important;
+}
+
+// Define classes for z-index
+$indexes: () !default;
+$indexes: map-merge(
+  (
+    0: 0,
+    1: 200,
+    2: 400,
+    3: 600,
+    4: 800,
+    5: 1000,
+    6: 1200,
+    7: 1400,
+    8: 1600,
+    9: 1800,
+    10: 2000
+  ),
+  $indexes
+);
+
+@each $index, $size in $indexes {
+  .elevation-#{$index} {
```

Comment:
One possible concern with the `.elevation-` class name is how it might cause confusion with elevation as it relates to shadows in the design system, for example:

<img width="732" alt="image" src="https://user-images.githubusercontent.com/2828721/159003744-67d880a9-401c-40e4-9933-08e2ffd91c7f.png">

z-index feels slightly different than "elevation" in this sense. Perhaps the CSS utility class is `.zindex-`?